client/src/components/PrevButton.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Button from "./shared/Button";
+
+function PrevButton(props) {
+    return (
+        <Button value="Prev" className="btn-secondary text-white" onClick={props.onClick} />
+    );
+}
+
+export default PrevButton;
```

Comment:
Separate Next/Prev button components seem to be a lot of overhead for minimal benefit in this case.

Reply:
I completely agree to this point. The purpose to create separate button components was let’s say in future if the button needs to be changed or any other thing needs to be added like tracking those button actions we would not require to update our actual carousel rendering file. Otherwise I completely agree if it is only for the current use case.